source/JWT/Fido.JWT.Manager.pas

```diff
@@ -44,7 +45,7 @@   TJWTManager = class(TInterfacedObject, IJWTManager)
   public
     function VerifyToken(const CompactToken: string; const Secret: TJOSEBytes): TJWT;
 
-    function GenerateToken(const Issuer: string; const DefaultValidityInSecs: Integer): TJWT;
+    function GenerateToken(const Issuer: string; const DefaultValidityInSecs: Extended = System.Math.Infinity): TJWT;
```

Comment:
Do you have updated tests for this to check if the infinite value is used?

Reply:
I did now.... daaahhhh